src/cordova/apple/xcode/ios/Outline/Outline-Info.plist

```diff
@@ -67,5 +73,9 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>UIDeviceFamily</key>
```

Comment:
What is this?

Reply:
It sounds like we would want to "Optimize interface for Mac" though, since it's a macos app, not iPad.